src/recipe.rs

```diff
@@ -193,7 +200,9 @@ impl<'src, D> Recipe<'src, D> {
 
       if config.dry_run
         || config.verbosity.loquacious()
-        || !((quiet_command ^ self.quiet) || config.verbosity.quiet())
+        || !((quiet_command ^ self.quiet)
+          || (context.settings.quiet && self.no_quiet())
+          || config.verbosity.quiet())
```

Comment:
I think this should be something like:

```suggestion
        || !((quiet_line ^ self.quiet) || (context.settings.quiet && !self.no_quiet()) || config.verbosity.quiet())
```

Reply:
done